docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,121 @@
+  
+   {{< note >}}
+   Disabled policy cannot be executed.
+   {{< /note >}}
+  
+1. Click **Edit** to open the edit dialog and do modifications to the selected policy.
+1. Click **Delete** to delete the selected policy.
+  
+   {{< note >}}
+   If the executions of the selected policy are still in progress, the deletion will be rejected.
+   {{< /note >}}
+
+## Manage Executions of Preheat Policy
+
+1. Select the policy by clicking the radio button at the front of the row. If the policy has been executed before, the 
+relevant executions will be listed in the execution data grid.
+   ![policy execution](../../../img/p2p-preheat/policy-execution.png)
+1. For each execution, you can find the following data:
+    - ID: identity of the execution with a hyperlink pointing to the detailed page
+    - Status: `Success`,`Error` and `Running`
+    - Trigger: the trigger way of the execution, it can be `Manual`,`Scheduled` and `Event-based`
+    - Start Time: the start time of the execution (rendered as local time format)
+    - Duration: the overall duration of the execution
+    - Success Rate: each execution may contain multiple tasks, the percent of the success ones over the total
+    
+   {{< note >}}
+   For the `Error` status, there will be a small info icon with tooltip that containing the error message next to it.
+   For the `Success` status, if there are no images matching the filters and criteria defined in the policy, a small 
+   info icon with tooltip which indicates that no images to preheat will be placed next to it.
+   {{< /note >}}
+    
+1. Click the ID hyperlink to open the detailed page of the execution.
+   ![execution details](../../../img/p2p-preheat/execution-details.png)
+  
+   {{< note >}}
+   Each execution may preheat more than one matched images ,and each image will be preheated by one task separately. 
+   As a result, the execution may include more than one preheating tasks.
+   {{< /note >}}
+  
+1.  Besides the general info, you can also find a simple metrics grouped by the status of tasks:
+    - **SUCCESS**: how many tasks have been finished
+    - **FAILURE**: how many tasks are failed to complete
+    - **IN PROGRESS**: how many tasks are under running
+    - **STOPPED**: how many tasks have been stopped
+1. All the related tasks of the execution are listed in the task data grid. You can find more detailed info of the task:
+    - **Artifact**: which artifact is preheating
+    - **Status**: the status of this preheating task
+    - **Digest**: the digest of the preheating image
+    - **Type**: the artifact type of the preheating artifact
+    - **Start Time**: the start time of this preheating task
+    - **Duration**: the overall duration of this preheating task
+    - **Logs**: a hyperlink to open the task logs for checking more details of this preheating task
+    
+   {{< note >}}
+   Currently, preheat only support image and then the value of `Type` column is always `image`.
```

Comment:
Suggestion: "Harbor only supports preheating images so the value of `Type` will always be `image`."